litellm/llms/bedrock.py

```diff
@@ -15,38 +12,24 @@ def __init__(self, status_code, message):
             self.message
         )  # Call the base class constructor with the parameters it needs
 
+
 class AnthropicConstants(Enum):
     HUMAN_PROMPT = "\n\nHuman:"
     AI_PROMPT = "\n\nAssistant:"
 
+
 def init_bedrock_client(region_name):
-    import sys
     import boto3
-    import subprocess
-    try:
-        client = boto3.client(
-            service_name="bedrock",
-            region_name=region_name,
-            endpoint_url=f'https://bedrock.{region_name}.amazonaws.com'
-        )
-    except Exception as e:
-        try:
-            command1 = "python3 -m pip install https://github.com/BerriAI/litellm/raw/main/cookbook/bedrock_resources/boto3-1.28.21-py3-none-any.whl"
-            subprocess.run(command1, shell=True, check=True)
-            # Command 2: Install boto3 from URL
-            command2 = "python3 -m pip install https://github.com/BerriAI/litellm/raw/main/cookbook/bedrock_resources/botocore-1.31.21-py3-none-any.whl"
-            subprocess.run(command2, shell=True, check=True)
-
-            import boto3
-            client = boto3.client(
-                service_name="bedrock",
-                region_name=region_name,
-                endpoint_url=f'https://bedrock.{region_name}.amazonaws.com'
-            )
-        except Exception as e:
-            raise e
+
+    client = boto3.client(
+        service_name="bedrock-runtime",
+        region_name=region_name,
+        endpoint_url=f'https://bedrock-runtime.{region_name}.amazonaws.com'
+    )
```

Comment:
The `invoke_model` method only exists in the `bedrock-runtime`, see the https://github.com/boto/boto3/issues/3881

Reply:
if possible can you run the bedrock tests in test_completion.py with these changes for bedrock and send a screenshot that they work. 

If not I can test them tomorrow

Reply:
The Titan test fails because I don't have access, but the rest pass. Please refer to the screenshots. Thank you for your prompt assistance! 
<img width="1918" alt="image" src="https://github.com/BerriAI/litellm/assets/3363189/c3f3cd71-d4e5-4cd7-97dd-becd054763ab">

Reply:
looks awesome - what is that tool you're using to run pytest. looks super cool

Reply:
😊 It's PyCharm with the following config: 
<img width="1041" alt="image" src="https://github.com/BerriAI/litellm/assets/3363189/0a3735aa-84ed-481e-a73d-1c632889a501">